jruby1/plan.sh

```diff
@@ -11,10 +11,10 @@ pkg_deps=(
   core/bash
   core/coreutils
   core/glibc
-  core/jre8
+  core/corretto8
 )
 pkg_build_deps=(
-  core/jdk8
+  core/corretto8
```

Comment:
This _shouldn't_ be necessary, as it's included in the `pkg_deps`.  Habitat build, when it is resolving dependencies, will automatically include runtime dependencies paths into the buildtime environment.

Reply:
Hi @smacfarlane.  Removed the duplicate dependency.